data-sources/azure-monitor/config.yml

```diff
@@ -0,0 +1,9 @@
+id: azure-monitor
```

Comment:
We already have an Azure data source, could we possibly use that?

Reply:
That data source is a different method for reporting Azure data, this is a new way.